tsconfig.json

```diff
@@ -3,8 +3,7 @@
     "target": "ES2018",
     "module": "commonjs",
     "lib": ["es2018"],
-    "declaration": false,
```

Comment:
Can you add declaration back, just with `true` instead of false.

Reply:
This has been done